docs/crio.8.md

```diff
@@ -233,7 +233,7 @@ For the bind-mount conditions, only mounts explicitly requested by Kubernetes co
 
 **--grpc-max-send-msg-size**="": maximum grpc receive message size (default: 16 * 1024 * 1024)
 
-**--host-ip**="": host IP considered as the primary IP to use by CRI-O for things such as host network IP (default: "")
+**--host-ip**="": Host IPs are the addresses to be used for sandboxes with specified host network, can be specified multiple times (default: [])
```

Comment:
It might be me pre-tea, but I'm not getting what you're trying to say here.  This is the first and only reference to "sandboxes" on this page.  Maybe:  "Host IPs are the addresses  to be used for the host network and can be specified multiple times (default: [])"

Reply:
Yes, I think it would be better to rephrase it, I applied your suggestion.